### PR TITLE
Make disk size configurable

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

**Comment by dlorenc** [comment]
Nice! I'm working on setting up automated integration tests.

@minikube-bot add to whitelist

**Comment by jimmidyson** [comment]
Excellent!

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -153,5 +155,6 @@ func init() {
 	startCmd.Flags().StringVarP(&vmDriver, "vm-driver", "", constants.DefaultVMDriver, fmt.Sprintf("VM driver is one of: %v", constants.SupportedVMDrivers))
 	startCmd.Flags().IntVarP(&memory, "memory", "", constants.DefaultMemory, "Amount of RAM allocated to the minikube VM")
 	startCmd.Flags().IntVarP(&cpus, "cpus", "", constants.DefaultCPUS, "Number of CPUs allocated to the minikube VM")
+	startCmd.Flags().IntVar(&disk, "disk-size", constants.DefaultDiskSize, "Disk size in gigabytes allocated to the minikube VM")
```
Comment (jimmidyson) [inline cmd/minikube/cmd/start.go]: Wonder if this should be a flag that takes units, e.g. 20G, rather than just an int?

**Comment by dlorenc** [comment]
Looks like this failed in Darwin with:

`_gopath/src/k8s.io/minikube/pkg/minikube/cluster/cluster_darwin.go:64: cannot use config.DiskSize (type int) as type int64 in field value
`

**Comment by jimmidyson** [comment]
@dlorenc Link to failed build log isn't public - guess you know about that?

**Comment by dlorenc** [comment]
Yeah, working on that :)

**Comment by jimmidyson** [comment]
@dlorenc Any reason why Jenkins Cross Build hasn't set status properly?

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (jimmidyson) [inline cmd/minikube/cmd/start.go]: Updated to use `unitsValue` type for flag using Docker units for parsing.

**Comment by dlorenc** [comment]
> @dlorenc Any reason why Jenkins Cross Build hasn't set status properly?
Sorry, just ignore it for now :) Still working out bugs in the plugin.

**Comment by dlorenc** [comment]
This LGTM. Thanks!

**Pull request merged**